app/models/like.rb

```diff
@@ -0,0 +1,11 @@
+class Like < ApplicationRecord
+  belongs_to :post
```

Comment:
* Kindly add the association between likes and users HINT 💭 since a user has many likes then a like belongs to a user eg (`belongs_to :author, class_name: 'User'`)